eZ/Publish/API/Repository/NotificationService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
+
+interface NotificationService
+{
+    /**
+     * Get currently logged user notifications.
+     *
+     * @param int $limit Number of notifications to get
+     * @param int $page Notifications pagination
+     *
+     * @return \EzSystems\Notification\SPI\Persistence\ValueObject\Notification[]
+     */
+    public function getUserNotifications($limit, $page);
+
+    /**
+     * Mark notification as read so it no longer bother the user.
+     *
+     * @param int $notificationId Notification id to be marked as read
+     */
+    public function markNotificationAsRead($notificationId);
```

Comment:
IMHO this method should take Notification object as an argument instead of id. There is also missing list of exceptions thrown by this method.